tests/foreman/cli/test_contentview.py

```diff
@@ -3341,6 +3341,53 @@ def test_positive_validate_force_promote_warning(self, target_sat, function_org)
         ]
         assert lce_prod['id'] == promoted_lce['id']
 
+    def test_cv_lce_order(session, target_sat, function_org):
+        """Verify that LCEs are displayed in Path order in the UI
+
+        :id: 44469755-69b5-4b95-a3c0-5746d71d5d1e
+
+        :steps:
+            1. Create multiple LCEs in a specific path order.
+            2. Create and promote a CV to all the LCEs.
+            3. Check hammer content-view version list --content-view-id <cv-id>
+
+        :expectedresults: The LCEs are displayed in path order.
+
+        :Verifies: SAT-28538
+
+        :customerscenario: true
+
+        :CaseImportance: High
+        """
+        # Create 4 LCEs prior one to each other
+        lces_list = []
+        for i in range(4):
+            if i == 0:
+                lces_list.append(
+                    target_sat.cli_factory.make_lifecycle_environment(
+                        {'organization-id': function_org.id}
+                    )
+                )
+            else:
+                lces_list.append(
+                    target_sat.cli_factory.make_lifecycle_environment(
+                        {'organization-id': function_org.id, 'prior': lces_list[i - 1].name}
+                    )
+                )
```

Comment:
Just a suggestion I couldn't resist, not tested myself. Feel free to ignore, result should be same.
```suggestion
        lces_list = [
            target_sat.cli_factory.make_lifecycle_environment({'organization-id': function_org.id})
        ]
        for _ in range(3):
            lces_list.append(
                target_sat.cli_factory.make_lifecycle_environment(
                    {'organization-id': function_org.id, 'prior': lces_list[-1].name}
                )
            )
```